lib/db/models/Node.ts

```diff
@@ -20,7 +20,24 @@ export default (sequelize: Sequelize.Sequelize, DataTypes: Sequelize.DataTypes)
         }
       },
     },
-    banned: { type: DataTypes.BOOLEAN, allowNull: true },
```

Comment:
I also think we can keep the banned flag on the nodes table. I figure this can be an easy way to check if a node is banned without even having to load and calculate their reputation upon startup.